tests/dummy/app/controllers/bars.js

```diff
@@ -22,7 +22,7 @@ export default Ember.Controller.extend({
   }),
 
   tableContent: Ember.computed(function() {
-    return _.range(100).map(function(index) {
+    return Array.apply(null, new Array(100)).map(function(_, index) {
```

Comment:
Can't do that unfortunately, see http://stackoverflow.com/questions/3895478/does-javascript-have-a-method-like-range-to-generate-an-array-based-on-suppl. But we could use a for loop to make it easiest to read. Since they're examples, I think I will do that.